java/Windowing/pom.xml

```diff
@@ -61,6 +61,11 @@
             <artifactId>flink-connector-kafka</artifactId>
             <version>${flink.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-connector-kinesis</artifactId>
```

Comment:
We should use 1.15.4 for the kinesis connector only, due to a known bug
Same thing we are doing [here](https://github.com/aws-samples/amazon-managed-service-for-apache-flink-examples/blob/d681495043a2a6b2e0c0f484ce907b4eaf3186be/java/KinesisConnectors/pom.xml#L19)